src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/methodparampad/InputWhitespaceAround.java

```diff
@@ -0,0 +1,31 @@
+package com.puppycrawl.tools.checkstyle.checks.whitespace.methodparampad;
+
+@SuppressWarnings({"this", "that"})
+public class InputWhitespaceAround
```

Comment:
Do you mean InputWhitespaceAround should be named InputMethodParamPadWhitespaceAround?

We split checks into separate files,so even if 2 checks have the same name input file they wouldn't depend on 1 file because they are in different namespaces